contrib/pp_osx_codesign_fix/pp_osx_codesign_fix.c

```diff
@@ -0,0 +1,96 @@
+/* -*- C -*- main.c */
+
```

Comment:
Please add a comment containing the command to build the pp_osx_codesign_fix executable.